static/home/css/style.css

```diff
@@ -492,6 +492,10 @@ nav{
 	transform: rotateX(360deg) translateX(-10px) translateY(-300px);
 	background-color: transparent;
 }
+.glide__bullets{
+	position: relative  !important;
+	padding:0 50% 0 50% !important;
```

Comment:
I believe that same effect can be achieved by just unsetting the `left` and `transform` property of `.glide_bullets` instead of adding additional `padding` in it.

Reply:
Ok, I will try to do with that method.